modules/environment-roles/templates/shared_terraform_policy_2.json.tpl

```diff
@@ -17,9 +17,10 @@
       ],
       "Resource" : [
           "arn:aws:ssm:eu-west-2:${account_id}:parameter/mgmt/cost_centre",
-          "arn:aws:ssm:eu-west-2:${account_id}:parameter/mgmt/trusted_ips",
           "arn:aws:ssm:eu-west-2:${account_id}:parameter/mgmt/external_ips",
-          "arn:aws:ssm:eu-west-2:${account_id}:parameter/mgmt/management_account"
+          "arn:aws:ssm:eu-west-2:${account_id}:parameter/mgmt/management_account",
+          "arn:aws:ssm:eu-west-2:${account_id}:parameter/mgmt/tna_account",
```

Comment:
I think "TNA account" is a bit ambiguous. Is this the organisational root account? It would be good to rename it so we don't confuse it with the mgmt account.

Reply:
What about "tna_root_account"?

Reply:
Maybe "tna_organisation_root_account"? It doesn't really matter if it's verbose because it's not going to be referenced in many places.